framework/physics/solver.cc

```diff
@@ -212,4 +215,19 @@ Solver::SetProperties(const ParameterBlock& params)
   }
 }
 
+void
+Solver::SetAuxiliaryFieldFunction()
+{
+  for (const auto& auxvar : auxvars_)
+  {
+    for (auto& ff : field_function_stack)
+    {
+      if (auxvar == ff->TextName())
```

Comment:
in the FF, we still use TextName(). Should it be changed to Name() later or now?

Reply:
This may have been changed in my previous PR #333 and need to be updated here.